install_config/configuring_authentication.adoc

```diff
@@ -947,15 +947,15 @@ link:#mapping-identities-to-users[as described above].
 <5> The client ID of a
 link:https://github.com/settings/applications/new[registered GitHub OAuth
 application]. The application must be configured with a callback URL of
-`<master>/oauth2callback/<identityProviderName>`.
+*_<master>/oauth2callback/<identityProviderName>_*.
```

Comment:
@ahardin-rh I'd leave the markup for the two callback URLs in this PR as-is (just backticks), since they're not really filenames/paths, but also because there's at least one other in this topic (elsewhere from this PR) that would now be different from these 2.